xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
+          configuration file. </para>
+        </callout>
+        <callout arearefs="co.rsync.conf.include">
+         <para> Loads any modules (or targets) from
+           <filename>/etc/rsyncd.d/*.conf</filename> files. These files should
+          not contain any references to global values. </para>
+        </callout>
+       </calloutlist>
+      </step>
+      <step>
+       <para> Create your module (your backup target) in the file
+         <filename>/etc/rsyncd.d/backup.conf</filename> with the following
+        lines: </para>
+       <screen># backup.conf: backup module
+[backup] <co xml:id="co.rsync.conf.target"/>
+   uid = &exampleuser_plain; <co xml:id="co.rsync.conf.uid-gid"/>
+   gid = users <xref linkend="co.rsync.conf.uid-gid"/>
+   path = /var/backup <co xml:id="co.rsync.conf.path"/>
+   auth users = &exampleuser_plain;  <co xml:id="co.rsync.conf.authusers"/>
+   secrets file = /etc/rsyncd.secrets <co xml:id="co.rsync.conf.secretfiles"/>
+   comment = Our backup target</screen>
+       <calloutlist>
+        <!--<callout arearefs="co.rsync.conf.usechroot">
+         <para>
+          Set protection against security holes. Before starting the transfer,
+          the Rsync daemon will change to this path (<quote>change root</quote>).
+         </para>
+        </callout>-->
+        <!--<callout arearefs="co.rsync.conf.logging">
+         <para>
+          Enables logging and store it in <filename>/var/log/rsyncd.log</filename>
+          with the format of <varname>log&nbsp;format</varname>.
+         </para>
+        </callout>-->
+        <callout arearefs="co.rsync.conf.target">
+         <para>
+          The <emphasis>backup</emphasis> target. You can invent any name you
+          like. However, it is a good idea to name a target according to its
+          purpose and store it in the <filename>*.conf</filename> file
+          respectively.
+         </para>
+        </callout>
+        <callout arearefs="co.rsync.conf.uid-gid">
+         <para>
+          Specifies the user name or group name that is used when the file
+          transfer takes place.
+         </para>
+        </callout>
+        <callout arearefs="co.rsync.conf.path">
+         <para>
+          Defines the path to store your backups.
+         </para>
+        </callout>
+        <callout arearefs="co.rsync.conf.authusers">
+         <para>
+          Specifies a comma separated list of allowed users. In its simplest
+          form, it lists the user names that are allowed to connect to this
+          module.
+         </para>
+        </callout>
+        <callout arearefs="co.rsync.conf.secretfiles">
+         <para>
+          Specifies the path of a file that contains lines with user names
+          and plain passwords.
+         </para>
+        </callout>
+       </calloutlist>
+      </step>
+      <step>
+       <para>
+        Create the file <filename>/etc/rsyncd.secrets</filename> with the
```

Comment:
Create the file <filename>/etc/rsyncd.secrets</filename>

**Create the <filename>/etc/rsyncd.secrets</filename> file**